lib/challenge.js

```diff
@@ -48,9 +67,15 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet; {
+
+  petName : "Frankie",
+  petSpecies : "Dog",
+  petBreed : "Min Pin",
+  petNoise : "Bark"
+};
+
```

Comment:
This is a variable declaration (end statement), object literal (end statement). There is no assignment going on here, so the object reference is immediately lost. The requested method is also missing. Your object literal is well-formed, though. Half credit.